src/Tizen.Location.Geofence/doc/api/Tizen.Location.Geofence.md

```diff
@@ -4,6 +4,10 @@ summary: The Tizen.Location.Geofence namespace provides classes for virtual peri
 remarks: *content
 ---
 ## Overview
+
+[!NOTE]
+All geofence APIs have been deprecated since Tizen 8.0 and will be removed after two releases without any alternatives.
```

Comment:
That note could says that whole Tizen.Location is deprecated, as rest already was marked with Obsolete. And we need `>` for the note to be displayed correctly in the old docfx we are using.
```suggestion
> [!NOTE]
> All Tizen.Maps APIs have been deprecated since Tizen 8.0 (API level 11) and will be removed after two releases.
```